tests/unit/components/notifications/HdNotificationsBar.spec.js

```diff
@@ -1,57 +1,70 @@
-import { wrapperFactoryBuilder } from 'tests/unit/helpers';
+import { shallowMount } from '@vue/test-utils';
```

Comment:
Why did you rollback from the `wrapperFactory`?

Reply:
Because I don't need any defaults in those tests.

Reply:
I've checked the other specs and seems like you've switched to using the wrapper factory in all of them, even in the cases where we don't actually need defaults.
I'm definitely for the idea of unifying the way we write tests, and also in favor of using the factory approach as I mentioned quite some time ago in this PR:
https://github.com/homeday-de/homeday-blocks/pull/53

I'll switch to using the factory here as well. And let's bring up this topic to the team sync, to get the team's thoughts.